compiler/sem.nim

```diff
@@ -28,6 +28,9 @@ when not defined(leanCompiler):
 
 # implementation
 
+proc isMacroRealGeneric*(s: PSym): bool {.importc.}
+proc semAlias2(c: PContext, n: PNode): PNode {.exportc.}
```

Comment:
We don't use ``importc`` and ``exportc`` to circumvent the cyclic import restrictions.

Reply:
we don't but we should, it's much better than relying on include (eg `include semtypes, semtempl, semgnrc, semstmts, semexprs`), and also better for incremental compilation (less tight coupling);

but for the sake of focusing on this PR I've addressed your comment and removed importc/exportc